Add Elaine Wang profile to team page (#77)
UCD-SERG/ucd-serg.github.io: people/yxewang/bio-yxewang.qmd

---
title: "Elaine Yuxuan Wang"
lastname: "Wang"
image: yxewang.jpg
image-alt: "Portrait photo of Elaine Yuxuan Wang"
about:
  template: solana
subtitle: "MS Student, Graduate Group in Applied Mathematics"
institution: "UC Davis"
---

> Elaine is a 1st year MS student in the Applied Mathematics program at UC Davis. Her current research focuses on developing the `serodynamics` R package, to support broader seroepidemiological modeling of  post-infection antibody dynamics. Her research interests lie at the intersection of statistical and mathematical modeling and infectious disease epidemiology, with a focus on disease forecasting and prevention in global health settings.

Reviewer: ```suggestion
Elaine is a 1st year MS student in the Applied Mathematics program at UC Davis. Her current research focuses on developing the `serodynamics` R package, to support broader seroepidemiological modeling of post-infection antibody dynamics. Her research interests lie at the intersection of statistical and mathematical modeling and infectious disease epidemiology, with a focus on disease forecasting and prevention in global health settings.
```

Author: addressed

> Elaine is a 1st year MS student in the Applied Mathematics program at UC Davis. Her current research focuses on developing the `serodynamics` R package, which extends the Serocalculator framework to support broader seroepidemiological modeling of antibody dynamics. Her research interests lie at the intersection of statistical and mathematical modeling and infectious disease epidemiology, with a focus on disease forecasting and prevention in global health settings.

Reviewer: ```suggestion
Elaine is a 1st year MS student in the Applied Mathematics program at UC Davis. Her current research focuses on developing the `serodynamics` R package, to support broader seroepidemiological modeling of  post-infection antibody dynamics. Her research interests lie at the intersection of statistical and mathematical modeling and infectious disease epidemiology, with a focus on disease forecasting and prevention in global health settings.
```

Email: yxewang@ucdavis.edu

> 

Reviewer: ```suggestion
possibly add github id
```

Author: addressed
